stixcore/ephemeris/manager.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, meta_kernel_path):
 
         with self.meta_kernel_path.open('r') as mk:
             original_mk = mk.read()
-            kernel_dir = str(self.meta_kernel_path.parent.parent.resolve())
+            kernel_dir = str(self.meta_kernel_path.parent.resolve())
```

Comment:
hopefully we should be able to replace all these with the imported paths so something like:
```python
from stixcore.data import test
...
kernel_dir  = str(test.ephemeris.META_KERNEL)
```
maybe for another issue?

Reply:
i changed that